Detector/DetSensitive/src/SimpleCalorimeterSD.cpp

```diff
@@ -10,6 +10,9 @@
 // CLHEP
 #include "CLHEP/Vector/ThreeVector.h"
 
+// Geant4
+#include "G4SDManager.hh"
+
 namespace det {
 SimpleCalorimeterSD::SimpleCalorimeterSD(const std::string& aDetectorName,
   const std::string& aReadoutName,
```

Comment:
can you initialize all members? m_calorimeterCollection seems missing for example.